src/components/event-form/event-form-thumbnail/event-form-thumbnail.tsx

```diff
@@ -49,7 +49,10 @@ export const EventFormThumbnail = ({ form, ref }: EventFormThumbnailProps) => {
 				<div className="relative col-span-1">
 					<EventFormThumbnailRightSectionDecorations />
 					{values.withPartners && values.partners?.length > 0 && (
-						<EventFormThumbnailSponsors partners={values.partners} />
+						<EventFormThumbnailSponsors
+							partners={values.partners}
+							partnerType={values.partnerType}
+						/>
```

Comment:
Nie podoba mi się nazwa tego komponentu: "Sponsors". Wyszło, że możemy mieć albo sponsorów, albo partnerów. Może powinniśmy zrobić bardziej ogólne określenie dla tych dwóch słów, np "Supporters"?
```suggestion
						<EventFormThumbnailSupporters
							supporters={values.partners}
							supporterType={values.partnerType}
						/>
```

Tak samo zmieniłbym nazewnictwo w  formularzu na Supporters oraz w Enumie, bo teraz mieszamy nazwy: raz używamy słowa Partner, a raz Sponsor, a tutaj koniec końców chodzi o to samo (o Supportera).